corehq/util/couchdb_management.py

```diff
@@ -59,5 +65,11 @@ def get_db_for_class(self, klass):
     def get_db_for_doc_type(self, doc_type):
         return Database(self.get_db_uri_for_doc_type(doc_type))
 
+    def get_db_for_postfix(self, postfix):
+        return Database(self.all_db_uris_by_slug[postfix])
```

Comment:
(it feels pretty weird to me that `get_db` creates it, but i guess that was just to mimic expected behavior elsewhere)